core/src/test/java/com/linecorp/armeria/client/RedirectSignatureTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.client.RedirectingClient.RedirectSignature;
+import com.linecorp.armeria.common.HttpMethod;
+
+public class RedirectSignatureTest {
```

Comment:
`public` is redundant in JUnit 5.

```suggestion
class RedirectSignatureTest {
```